apps/main/e2e/orders.spec.ts

```
@@ -13,7 +13,7 @@ async function navigateToOrders(page: Page) {
  * Orders page (pro-only).
  */
 test.describe('Orders Page (Pro)', () => {
-	test.beforeEach(async (_, testInfo) => {
+	test.beforeEach(async ({}, testInfo) => {
```

Biome (2.3.13)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)